lua/core/default_config.lua

```diff
@@ -0,0 +1,221 @@
+-- IMPORTANT NOTE : This is default config, so dont change anything here.
+-- use custom/chadrc.lua instead
+
+local M = {}
+M.options, M.mappings, M.plugin = {}, {}, {}
+
+-- non plugin normal, available without any plugins
+M.options = {
+   -- NeoVim/Vim options
+   clipboard = "unnamedplus",
+   cmdheight = 1,
+   ruler = false,
+   hidden = true,
+   ignorecase = true,
+   mapleader = " ",
+   mouse = "a",
+   number = true,
+   -- relative numbers in normal mode tool at the bottom of options.lua
+   numberwidth = 2,
+   relativenumber = false,
+   expandtab = true,
+   shiftwidth = 2,
+   smartindent = true,
+   tabstop = 8, -- Number of spaces that a <Tab> in the file counts for
+   timeoutlen = 400,
+   -- interval for writing swap file to disk, also used by gitsigns
+   updatetime = 250,
+   undofile = true, -- keep a permanent undo (across restarts)
+   -- NvChad options
+   nvChad = {
+      copy_cut = true, -- copy cut text ( x key ), visual and normal mode
+      copy_del = true, -- copy deleted text ( dd key ), visual and normal mode
+      insert_nav = true, -- navigation in insertmode
+      window_nav = true,
+      theme_toggler = false,
+      -- used for updater
+      update_url = "https://github.com/NvChad/NvChad",
```

Comment:
as mentioned in the description of this PR that the updater.lua will no longer work , so why is this update url needed?

Reply:
Not exactly,

`updater.lua` will need to be slightly changed, as we are moving `chadrc` to lua/custom/`.

this is why.